docs/index.css

```diff
@@ -121,7 +120,7 @@ thead th:first-child {
 }
 
 code {
-  font-family: Monaco, Consolas, 'Lucida Console', monospace;
+  font-family: Monaco, Consolas, "Lucida Console", monospace;
```

Comment:
I thought the convention was single quotes – can you check if there is a Sass or Prettier configuration issue?

Reply:
@BPScott updated the prettier config to not format these files, so not an issue.

Ben, what was your rationale there?

Reply:
This was deliberatly changed - The docs folder is [no longer formatted with prettier](https://github.com/Shopify/polaris-tokens/pull/105/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R17) (but [I do use sass's expanded style](https://github.com/Shopify/polaris-tokens/pull/105/files#diff-b9e12334e9eafd8341a6107dd98510c9R253) to maintain most of the readable formatting  :)

I split the build into "legacy", "modern" and "docs" steps, this makes it easy to do just the modern build in shipit (to generate the `dist-modern` folder), and also change the docs build to something new (probably a proper gatsby app) in the near future. In order to retain the old behaviour I would have had to either:

- have two steps "build-legacy:format" (to format the dist folder with prettier) and "docs:format" (to format the css in the docs folder with prettier) to that run after each of the build-legacy and docs steps
- A single step that runs after both "build-legacy" and "docs" - which makes makes the of steps a bit more convoluted and less self-contained.

I figured seeing as this file is generated content, and is going away imminently it is easier to do create a build-legacy:format step that formats just the dist folder, and skip formatting the docs folder css with prettier, as sass already does a pretty solid job. Incidentally I've just realised by formatting this file after sass does its thing we risk breaking the css.map file that the sass build generates (as that maps locations in the not-prettiered css file, not the version that's been ran through prettier)